testing/src/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`App Component should have hello world message 1`] = `
+<div>
+  <h1>
+    Hello World
+  </h1>
+  <div>
+    <h2>
+      Review Section
+    </h2>
+    <h3>
+      These are hard coded reviews.
+    </h3>
+    <div>
+      <ul>
+        <li>
+          Comfortable and practical.
+        </li>
+        <li>
+          They are very dark. But that's good because I'm in very sunny spots
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+`;
```

Comment:
I'm not familiar with Jest snapshot, so I'm not quite sure what I'm looking at here...my best guess is that these are files that get outputted from running the tests. If that's the case and we want to use these, I'm assuming this whole folder should not be checked into git, and should be added to the git ignore